demos/py_scripts/gromacs.py

```diff
@@ -38,20 +24,23 @@
 
 
 def main(_):
-    """Run a Gromacs energy minimization using user-provided input files."""
+    """Run a Gromacs command."""
+
+    inductiva.api_key = "6e2a3936beba573da6c4f81b23b2650395c6911633397ee9dec8307b51c5f451"
```

Comment:
remove the key!

Reply:
and it's not enough to remove it, because it stays here on github forever, you need to invalidate this one, and regenerate a new one for you.